modules/utxo-lib/src/bitgo/wallet/Unspent.ts

```diff
@@ -0,0 +1,70 @@
+import { Network } from '../../networkTypes';
+import { UtxoTransactionBuilder } from '../UtxoTransactionBuilder';
+import { createOutputScript2of3, scriptTypeForChain } from '../outputScripts';
+import { toOutputScript } from '../../address';
+import { signInput2Of3, verifySignatureWithPublicKeys } from '../signature';
+import { WalletUnspentSigner } from './WalletUnspentSigner';
+import { RootWalletKeys } from './WalletKeys';
+import { UtxoTransaction } from '../UtxoTransaction';
+import { Triple } from '../types';
+import { toOutput, Unspent } from '../Unspent';
+
+export interface WalletUnspent {
```

Comment:
I figure this should extend/implement the `Unspent` interface and inherit the `id`/`address`/`value` fields.

Reply:
good idea